src/js_command_match/dev_deploy.rs

```diff
@@ -33,7 +33,7 @@ impl DevDeployArgs {
         if let Some(init_args) = &self.init_args {
             let mut initial_function = "new".to_string();
             if let Some(init_function) = &self.init_function {
-                initial_function = init_function.clone();
+                initial_function.clone_from(init_function);
             }
```

Comment:
This whole block can be removed and `initial_function` below can be replaced with:

```rust
self.init_function.as_ref().unwrap_or("new").to_owned()
```

Reply:
done